test/fixtures/dialects/postgres/postgres_create_role.yml

```diff
@@ -3,26 +3,189 @@
 # computed by SQLFluff when running the tests. Please run
 # `python test/generate_parse_fixture_yml.py`  to generate them after adding or
 # altering SQL files.
-_hash: 8e48269b8cd80d99bf7ffdeb0458e75ffdd8c46adfafb5848697400671b310c5
+_hash: 3c065b7b23271c44a013142a8e90b5338e344df2496bafe7752ec4b47a1a7eb0
 file:
+- statement:
+    create_user:
```

Comment:
Looks like you need to regenerate your YAML files since `create_user` no longer exists as a type now as these will be `create_role`.